release-calendar/test/test-data.ts

```diff
@@ -32,13 +32,12 @@ export default async function addTestData(
 ): Promise<void> {
   await Promise.all(
     [
-      new Release('1234567890123'), // lts
-      new Release('2234567890123'), // TODO(estherkim): promote to stable
-      new Release('3234567890123'), // TODO(estherkim): promote to percent_*
-      new Release('4234567890123'), // TODO(estherkim): promote to opt_in_*
-      new Release('5234567890123'), // nightly
-      new Release('6234567890123'),
-    ].map(async (release) => {
+      new Release('1234567890123'), // nightly
+      new Release('2234567890123'), // lts
+      new Release('3234567890123'), // perc-beta and perc-experimental
+      new Release('4234567890123'), // stable
+      new Release('5234567890123'), // opt-in-beta and opt-in-experimental
+    ].map(async release => {
       return repositoryService.createRelease(release);
```

Comment:
Can we stagger the create date so that all releases aren't created on the same day?

Reply:
Can do! Also this meant adding a optional date to the parameter of .createRelease which is pretty good to have, so yay for having to do this :)